packages/now-cli/test/dev/fixtures/monorepo-dynamic-paths/blog/next.config.js

```diff
@@ -0,0 +1,8 @@
+const assetPrefix = process.env.BUILDING_FOR_NOW ? '/blog' : '';
```

Comment:
Is `BUILDING_FOR_NOW` necessary for this test? I was thinking it failed regardless of `next.config.js`

Reply:
I lifted the example from the next.js repository instead of my own because I felt there was less `noise` there this is straight from there ... 
since the test doesn't try to access `/blog/pages/index.js` I suppose i can remove it

Reply:
I dropped all the static assets, the next.config.js and the buid.env section from the integration test.